docs/program/extend/SDK-as-server.md

```diff
@@ -35,18 +28,26 @@ Once your new arm is registered, you will be able to use it remotely with Viam.
 Here is an example of [how to create a custom arm component in the Python SDK documentation](https://python.viam.dev/examples/example.html#subclass-a-component).
 {{% /alert %}}
 
-To create a custom component and connect it to the RDK:
+To add a custom resource as a remote:
+
+1. Subclass a component and implement desired functions.
 
-1. Subclass a component and implement desired functions
-    - If you are using the Python SDK, you must define all functions.
-      For functions you do not want to implement, put `pass` or `raise NotImplementedError()` in the function.
+    You must define all functions belonging to a built-in resource type if defining a new model of an existing type.
     Otherwise, the class won't instantiate.
-    - If you are using the Go SDK, you must define all functions but you can leave the ones you do not wish to implement empty.
-2. Create an `rpc.server.Server` instance and register the custom component.
-3. Start the server and register the running server as a remote.
+    If you are using the Python SDK, put `pass` or `raise NotImplementedError()` in the body of functions you do not want to implement.
```

Comment:
All this language is already python-sdk specific, so "if you're using the python sdk" should probably come sooner. The process for doing this in go uses different terminology and paradigms.